test/helpers.js

```diff
@@ -25,47 +25,41 @@ const pinoFactory = (opts) => {
 
 const serverFactory = (messages, opts, fastifyOpts) => {
   const destination = new Writable({
-    write (chunk, enc, cb) {
+    write (chunk, _enc, cb) {
       messages.push(chunk.toString())
 
       process.nextTick(cb)
     }
   })
 
-  const pino = pinoFactory({ destination, ...opts })
+  const pinoLogger = pinoFactory({ destination, ...opts })
 
   return fastify({
-    logger: pino,
+    logger: pinoLogger,
     ...fastifyOpts
   })
 }
 
+const DateOriginalNow = Date.now
```

Comment:
maybe `dateOriginalNow` or `DATE_ORIGINAL_NOW`?

Reply:
I dont think it make a big difference.

Reply:
Yes agreed that it doesn’t make a big difference!

Just trying to provide my two cents here cuz usually only namespaces or classes use an upper camel case naming in JS.

Reply:
I understand. I just found this issue when I did a sonarqube scan over the whole organization. I just wanted to fix that alert.

Reply:
fixed